controllers/GeneralService.php

```diff
@@ -29,4 +43,57 @@ function getXMLFromValidation($url, $paramsFromGet)
 
         return $xml;
     }
+
+    public function getXMLForAgent($url, $paramsForPost)
```

Comment:
This function is not about getting a XML for agent, its more a Curl post request ...

Reply:
fixed